flutter/test/default_integrations_test.dart

```diff
@@ -177,6 +179,21 @@ void main() {
         false);
   });
 
+  test('nativeSdkIntegration closes native SDK', () async {
+    final completer = Completer<bool>();
```

Comment:
this could be as simple as a boolean var, no need of a `Completer`, just a nit though, feel free to ignore it

Reply:
Nah, gotta fix it.